packages/analytics/src/rudderstack.ts

```diff
@@ -105,8 +105,8 @@ export class RudderStack {
     }
 
     init() {
-        const isProduction = process.env.CIRCLE_JOB === 'release_production';
-        const isStaging = process.env.CIRCLE_JOB === 'release_staging';
+        const isProduction = process.env.NODE_ENV === 'production';
```

Comment:
```suggestion
        const is_production = process.env.NODE_ENV === 'production';
```
Naming convention. Same as below